.github/ci/packages.apt

```diff
@@ -0,0 +1,19 @@
+binutils-dev
+libgflags-dev
+libignition-cmake2-dev
+libignition-common3-dev
+libignition-fuel-tools3-dev
+libignition-gazebo2-dev
+libignition-gui2-dev
+libignition-math6-dev
+libignition-msgs4-dev
+libignition-physics-dev
+libignition-plugin-dev
+libignition-sensors2-dev
+libignition-tools-dev
+libignition-transport7-dev
+libsdformat8-dev
+libtinyxml2-dev
+libwebsockets-dev
+libyaml-dev
+python-yaml
```

Comment:
Why is this needed?

Reply:
No idea, it was just propagated from bitbucket pipelines:

https://github.com/ignitionrobotics/ign-launch/blob/a284b73fae4c116529295ac2a84a990dfb405c1a/bitbucket-pipelines.yml#L17

I'll remove it and see what happens.

Reply:
It works without it: d85eb63